api/governance/models.py

```diff
@@ -0,0 +1,121 @@
+import datetime
+from decimal import Decimal
+import django
+
+from django.db import models
+from django.db.models import Sum
+from django.shortcuts import reverse
+
+from coin.models import Coin, Transfer
+
+from .utils import PROPOSAL_DURATION_DAYS, Vote
+
+
+class ProposalVote(models.Model):
+    VOTES = (
+        (Vote.FOR, 'For'),
+        (Vote.AGAINST, 'Against'),
+        (Vote.ABSTAIN, 'Abstain'),
+    )
+
+    voter = models.ForeignKey('user.User', on_delete=models.CASCADE)
+
+    vote = models.CharField(
+        max_length=255, choices=VOTES, default=Vote.ABSTAIN)
+    amount = models.DecimalField(max_digits=20, decimal_places=4, default=0)
+
+    created_at = models.DateTimeField(auto_now_add=True)
+
+    def __str__(self):
+        return f"{self.voter} - {self.vote} - {self.amount}"
+
+
+class Proposal(models.Model):
+    def save(self, *args, **kwargs):
+        if not self.closed_at:
+            self.closed_at = django.utils.timezone.now(
+            ) + datetime.timedelta(days=PROPOSAL_DURATION_DAYS)
+
+        super(Proposal, self).save(*args, **kwargs)
+
+    approved = models.BooleanField(default=False)
+
+    proposed_by = models.ForeignKey('user.User', on_delete=models.CASCADE)
+
+    title = models.CharField(max_length=255)
+    description = models.TextField()
+
```

Comment:
Proposing 2 more fields in this model. In the design, there is a summary field for the Proposal. Seen in a lot of similar governance implementations. Good to add textfield, maybe with a max length? 

There are also categorical tags (Dashboard, UX), not sure the best way to implement but a form of array would be nice ( since that's how its currently implemented on the front end ;) ). 

```suggestion
summary = models.TextField(max_length=255)
category_tags = ArrayField(models.CharField())

```
<img width="744" alt="Screen Shot 2022-08-10 at 10 33 49 AM" src="https://user-images.githubusercontent.com/33527785/183945733-554aa737-d848-4bbc-9562-baf24bae37d4.png">